packages/document/docs/en/shared/config/server/publicDir.md

```diff
@@ -0,0 +1,71 @@
+- **Type:** `string | false | object`
```

Comment:
I think we do not need to provide the `string` type, just `object | false` is enough.

This can make `mergeRsbuildConfig` works correctly by default.